src/__stories__/ScrollList/index.tsx

```diff
@@ -14,6 +14,7 @@ import { FixedSizeList, IFixedSizeListProps } from '../../ScrollList';
 export const fixedSizeListKnobs = (tabName = 'FixedSizeList'): Omit<IFixedSizeListProps, 'children'> => ({
   itemCount: number('itemCount', 20, { min: 0, max: Infinity, range: false, step: 1 }, tabName),
   itemSize: number('itemSize', 50, { min: 0, max: Infinity, range: false, step: 1 }, tabName),
+  offset: number('offset', 0, { min: 0, max: Infinity, range: false, step: 1 }, tabName),
```

Comment:
You can have a negative offset

```suggestion
  offset: number('offset', 0, { min: Infinity, max: Infinity, range: false, step: 1 }, tabName),
```